external_deps/build.sh

```diff
@@ -430,10 +543,11 @@ build_png() {
 	"${download_only}" && return
 
 	cd "${dir_name}"
-	# The default -O2 is dropped when there's user-provided CFLAGS.
-	CFLAGS="${CFLAGS} -O2" ./configure --host="${HOST}" --prefix="${PREFIX}" --libdir="${PREFIX}/lib" "${CONFIGURE_SHARED[@]}"
-	make
-	make install
+
+	cmake_build \
```

Comment:
There's a regression with the PNG build for MSVC. It produces an unwanted file at `prefix/lib/libpng.dll`. This is not a DLL, but a symlink to a mingw-format archive (useless for MSVC). In addition to this file being undesirable, something about the symlink causes tar to return an error code in the `package` step.